setup.cfg

```diff
@@ -32,9 +32,14 @@ exclude =
   # Trash and cache:
   .git
   __pycache__
-  .venv
   .eggs
   *.egg
+  # Environments
```

Comment:
After researching this topic a bit, I am going to reject this change. Here's why:
- `flake8` does not feature this on purpose: https://gitlab.com/pycqa/flake8/issues/579 and https://gitlab.com/pycqa/flake8/issues/36 There are two many possible ways to go wrong. There are also `.venv-SOME_HASH`, `.venv3.7`, `.venv3`, `.venv3.7.5`, etc

- It can be easily fixed on the users side. You can rename `venv` into `.venv` and done! `poetry` by default uses `.venv` when configured with `virtualenvs.in-project = true`

So, thanks a lot for this suggestion and these knew information I got!